client/my-sites/upgrades/controller.jsx

```diff
@@ -216,7 +216,7 @@ module.exports = {
 
 		analytics.pageView.record( basePath, 'Checkout Thank You' );
 
-		context.store.dispatch( setSection( 'checkout-thank-you', { hasSidebar: false } ) );
+		context.store.dispatch( setSection( { name: 'checkout-thank-you' }, { hasSidebar: false } ) );
```

Comment:
Selector seems to have been updated to this change. Nice.

https://github.com/Automattic/wp-calypso/blob/master/client/state/ui/selectors.js#L44